src/base/static/components/templates/right-sidebar.js

```diff
@@ -29,7 +29,7 @@ const RightSidebar = props => {
 
 RightSidebar.propTypes = {
   rightSidebarConfig: PropTypes.shape({
-    show: PropTypes.bool.isRequired,
+    enabled: PropTypes.bool.isRequired,
     visibleDefault: PropTypes.bool,
```

Comment:
Let's try to stick with the "is" prefix for bools, if possible. So these should be `isEnabled`, and `isVisibleDefault`.

Of course, no need to worry about it if it will cause a lot of config churn.